src/test/resources/GenerateWithWorkspaceTest/buf generate fails with no buf gen template file/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 Andrew Parmet
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id 'com.parmet.buf'
+}
+
+buf {
+  generate {
+    templateFileLocation = project.file("subdir/buf.gen.yaml")
```

Comment:
The equivalent test in GenerateTest has no template file location specified: https://github.com/andrewparmet/buf-gradle-plugin/blob/ce50a1c53e1805cf4cf1088c1cd519f9fd2047d2/src/test/resources/GenerateTest/buf%20generate%20fails%20with%20no%20buf%20gen%20template%20file/build.gradle

Rather than just making them consistent, I think it's worth testing both of these cases in both test classes: `buf generate fails with no default template file and no override specified` and `buf generate fails with a nonexistent specified template file`.

Reply:
Thanks for pointing out that discrepancy! Updated the PR to include the two new tests as requested.